assets/src/dashboard/templates/index.js

```diff
@@ -27,41 +27,127 @@ export default function (config) {
     {
```

Comment:
I was able to take the color values from our figma designs, but the color names, the `tags` and the `description` are _not_ on figma so I just made them up for now.  When we really dive into creating the templates, we can ask Sam for all the correct meta data per template.